src/ucp/rndv/rndv_ppln.c

```diff
@@ -174,21 +174,22 @@ ucp_proto_rndv_ppln_frag_complete(ucp_request_t *freq, int send_ack,
 
 void ucp_proto_rndv_ppln_send_frag_complete(ucp_request_t *freq, int send_ack)
 {
-    ucp_proto_rndv_ppln_frag_complete(freq, send_ack,
+    ucp_proto_rndv_ppln_frag_complete(freq, send_ack, 0,
                                       ucp_proto_request_complete_success,
                                       "ppln_send");
 }
 
 void ucp_proto_rndv_ppln_recv_frag_clean(ucp_request_t *freq)
 {
     ucp_proto_rndv_ppln_frag_complete(
-            freq, 0, (ucp_proto_complete_cb_t)ucs_empty_function,
+            freq, 0, 0, (ucp_proto_complete_cb_t)ucs_empty_function,
             "ppln_recv_clean");
 }
 
-void ucp_proto_rndv_ppln_recv_frag_complete(ucp_request_t *freq, int send_ack)
+void ucp_proto_rndv_ppln_recv_frag_complete(ucp_request_t *freq, int send_ack,
```

Comment:
maybe make this function inline?

Reply:
It's used in `rndv_rtr.c` and `rndv_get.c`, also it's aligned with `ucp_proto_rndv_ppln_send_frag_complete`. Moving `ucp_proto_rndv_ppln_recv_frag_complete` to `rndv_ppln.inl` will require more refactoring since `ucp_proto_rndv_ppln_frag_complete` is defined as a `static` in `rndv_ppln.c`.
So far the functions look to me in their correct place.

Reply:
ok